packages/timeline-state-resolver/src/service/device.ts

```diff
@@ -63,7 +63,7 @@ export interface BaseDeviceAPI<DeviceState, Command extends CommandWithContext>
 	 */
 	diffStates(oldState: DeviceState | undefined, newState: DeviceState, mappings: Mappings): Array<Command>
 	/** This method will take a command and send it to the device */
-	sendCommand(command: Command): Promise<any>
+	sendCommand(command: Command): Promise<void>
```

Comment:
`sendCommand` returns the result of the command in my design which is to be used in a generic handler for retries, media status and partial control. That means at some point I need these values to be non-void. Although using any is also not ideal here.

Reply:
I don't have a problem with them being non-void, but `any` is a bit of a smell and was unnecessary based on usage. 
So change it when you get to that retry handler, until then `void` is more descriptive of the current intention